dev/bootstrap.cfg

```diff
@@ -10,6 +10,7 @@ puppetlabs.services.legacy-routes.legacy-routes-service/legacy-routes-service
 puppetlabs.services.puppet-admin.puppet-admin-service/puppet-admin-service
 puppetlabs.trapperkeeper.services.authorization.authorization-service/authorization-service
 puppetlabs.services.versioned-code-service.versioned-code-service/versioned-code-service
+puppetlabs.trapperkeeper.services.scheduler.scheduler-service/scheduler-service
```

Comment:
Some tk deps are brought in using defs like this: https://github.com/puppetlabs/puppetserver/blob/8152bc8700009f96717d269958bf2b2fb569e7b1/test/unit/puppetlabs/services/config/puppet_server_config_service_test.clj#L21-L23. There may be more kicking around. That was the first I found in a search. Most have `deps` in the name.